src/test/java/duke/command/ExitCommandTest.java

```diff
@@ -0,0 +1,13 @@
+package duke.command;
+
+import duke.task.Todo;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+public class ExitCommandTest {
```

Comment:
```suggestion
import static org.junit.jupiter.api.Assertions.assertEquals;

public class ExitCommandTest {
```
Whitespace here too! :)